Makefile

```diff
@@ -40,7 +40,9 @@ checks:
 
 .PHONY: diff
 diff: node_modules
-	if (npx elm diff | tee /dev/stderr | grep -q MAJOR); then echo "MAJOR changes are not allowed!"; exit 1; fi
+	true
+# 	Temporarily disabled to allow for an intentional MAJOR upgrade
+# 	if (npx elm diff | tee /dev/stderr | grep -q MAJOR); then echo "MAJOR changes are not allowed!"; exit 1; fi
```

Comment:
❓  Does this mean we need to release this snapshot of the code and then make another commit reverting this change to the Makefile?

Reply:
Yes!